tests/base/test_regularization.py

```diff
@@ -39,6 +39,16 @@
     "BaseAmplitude",
     "VectorAmplitude",
     "CrossReferenceRegularization",
+    # Removed regularization classes that raise error on instantiation
+    "PGIwithNonlinearRelationshipsSmallness",
+    "PGIwithRelationships",
+    "Simple",
+    "SimpleSmall",
+    "SimpleSmoothDeriv",
+    "Small",
+    "SmoothDeriv",
+    "SmoothDeriv2",
+    "Tikhonov",
 ]
```

Comment:
I imagine this "IGNORE_ME" list is starting to include most of our regularizations.

Reply:
I suspect so... I don't quite like this list. I would prefer a more "pro-active" approach of listing which classes we want to test, instead of listing the ones we are going to ignore. Moreover, that list is only used in some tests: is not that all test functions `test_regularization` will not test those classes.

I'll try to improve that in a separate PR since it's not critical for this one.